sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyClient.cs

```diff
@@ -1106,5 +1111,110 @@ public virtual async Task<Response<KeyVaultKey>> ImportKeyAsync(ImportKeyOptions
+        /// <param name="environment">The target environment assertion.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>The <see cref="KeyVaultKey"/> that was exported along with the private key if exportable.</returns>
+        /// <exception cref="ArgumentException"><paramref name="name"/> or <paramref name="environment"/> is an empty string.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="name"/> or <paramref name="environment"/> is null.</exception>
+        /// <seealso cref="ExportKeyAsync(string, string, string, CancellationToken)"/>
+        public virtual async Task<Response<KeyVaultKey>> ExportKeyAsync(string name, string environment, CancellationToken cancellationToken = default) =>
+            await ExportKeyAsync(name, null, environment, cancellationToken).ConfigureAwait(false);
+
+        /// <summary>
+        /// Exports a specific version of a <see cref="KeyVaultKey"/> including the private key if originally created with <see cref="CreateKeyOptions.Exportable"/> set to true,
+        /// or imported with <see cref="KeyProperties.Exportable"/> in <see cref="ImportKeyOptions"/> set to true.
+        /// </summary>
+        /// <remarks>
+        /// Requires the <see cref="KeyOperation.Export"/> permission.
+        /// </remarks>
+        /// <param name="name">The name of the key to export.</param>
+        /// <param name="version">The optional version of the key to export.</param>
+        /// <param name="environment">The target environment assertion.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>The <see cref="KeyVaultKey"/> that was exported along with the private key if exportable.</returns>
+        /// <exception cref="ArgumentException"><paramref name="name"/> or <paramref name="environment"/> is an empty string.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="name"/> or <paramref name="environment"/> is null.</exception>
+        /// <seealso cref="ExportKey(string, string, CancellationToken)"/>
+        public virtual Response<KeyVaultKey> ExportKey(string name, string version, string environment, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(name, nameof(name));
+            Argument.AssertNotNullOrEmpty(environment, nameof(environment));
+
+            using DiagnosticScope scope = _pipeline.CreateScope($"{nameof(KeyClient)}.{nameof(ExportKey)}");
+            scope.AddAttribute("key", name);
+            scope.Start();
+
+            try
+            {
+                return _pipeline.SendRequest(RequestMethod.Post, new KeyExportParameters(environment), () => new KeyVaultKey(name), cancellationToken, KeysPath, name, "/", version, "/export");
```

Comment:
is it worth extracting `"/"` and `"/export"` into consts for both methods?

Reply:
Not really. We hadn't elsewhere, and consts will just be "stamped" in place by the compiler anyway. The compiled result is no different. I think this is more readable, and it's what generated code does as well (generated code uses very few constants).